auth/auth.go

```diff
@@ -32,10 +32,10 @@ const (
 	UserIDField keyType = iota
 )
 
-// HTTPAuthService defines how to handle requests for various http authentication methods.
+// HTTPAuthService defines how to handle requests for various http authn and authz methods.
```

Comment:
Can we spell out authn and authz in the code as well as in the comments? Would make them more readable.

Reply:
sure thing!

Reply:
Need to spell these out as well yeah?